pkg/securityscan/job/job.go

```diff
@@ -142,7 +142,21 @@ func New(clusterscan *cisoperatorapiv1.ClusterScan, clusterscanprofile *cisopera
 							HostPath: &corev1.HostPathVolumeSource{
 								Path: `/etc/group`,
 							},
+						},
+					}, {
+						Name: `journalctl`,
+						VolumeSource: corev1.VolumeSource{
+							HostPath: &corev1.HostPathVolumeSource{
+								Path: ` /var/log/journal`,
```

Comment:
Will this vary by distribution? (e.g. `/run/log/journal`)

Reply:
It will not since the docker image is using ubuntu 18:04 this will happen inside the sonobouy pod

Reply:
Sounds good. Should we add a check to ensure that the distribution is ubuntu-based then? Or is that out of scope?

Reply:
I'm confused, are we mounting the journal from the actual host, or from another outer container that we know will be Ubuntu?

Reply:
> Sounds good. Should we add a check to ensure that the distribution is ubuntu-based then? Or is that out of scope?

Nah, we control all of this process.

Reply:
Where are these used? By some test controls for k3s?

Reply:
Yes, basically for the journalctl command in the audits for k3s tests